src/app/static/src/js/components/runReport/runReportTabs.vue

```diff
@@ -63,11 +64,17 @@
         },
         methods: {
             setSelectedReportKey(e){
-                this.selectedRunningReportKey = e
+                this.selectedRunningReportKey = e;
+                session.setSelectedRunningReportKey(e);
             },
             switchTab(tab){
-                this.selectedTab = tab
+                this.selectedTab = tab;
+                session.setSelectedTab(tab);
             }
+        },
+        mounted(){
+            this.selectedTab = session.getSelectedTab() || "runReport";
+            this.selectedRunningReportKey = session.getSelectedRunningReportKey() || "";
```

Comment:
Not a big fan of the duplication with L61-62. Perhaps you could choose to only overwrite existing values if there is a value in session.